CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 ### Added
+- OxyPlot.Series.HeatMapSeries.GetValueForLabels (#256)
```

Comment:
Please disregard the two previous comments.

I found a superior fix for GetValue that eliminates the need to introduce GetValueForLabel.